main.go

```diff
@@ -1,16 +1,14 @@
 package main
 
 import (
-	"flag"
 	"log"
 	"net/http"
 
 	"gitlab.com/avarf/getenvs"
 )
 
 func main() {
-	flag.Parse()
-	http.Handle("/", http.FileServer(http.Dir(getenvs.GetEnvString("PATH", "/dev/shm"))))
+	http.Handle("/", http.FileServer(http.Dir(getenvs.GetEnvString("SERVER_PATH", "/dev/shm"))))
```

Comment:
suggestion to rename to FILESERVER_ROOT (or something related to fileserver name)